Index: src/main/resources/components/bankBox.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n\r\n<?import javafx.scene.layout.AnchorPane?>\r\n\r\n\r\n<AnchorPane maxHeight=\"-Infinity\" maxWidth=\"-Infinity\" minHeight=\"-Infinity\" minWidth=\"-Infinity\" prefHeight=\"400.0\" prefWidth=\"600.0\" xmlns=\"http://javafx.com/javafx/21\" xmlns:fx=\"http://javafx.com/fxml/1\" />\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/components/bankBox.fxml b/src/main/resources/components/bankBox.fxml
--- a/src/main/resources/components/bankBox.fxml	
+++ b/src/main/resources/components/bankBox.fxml	
@@ -1,6 +1,79 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.scene.layout.AnchorPane?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.BorderPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.text.Font?>
 
 
-<AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/21" xmlns:fx="http://javafx.com/fxml/1" />
+<fx:root fx:id="creditCard" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" type="BorderPane" xmlns="http://javafx.com/javafx/21" xmlns:fx="http://javafx.com/fxml/1">
+   <top>
+      <HBox spacing="200.0" BorderPane.alignment="CENTER">
+         <children>
+            <Label fx:id="bankNameLabel" text="Bank Name">
+               <font>
+                  <Font size="20.0" />
+               </font>
+            </Label>
+            <Label text="Credit Card">
+               <font>
+                  <Font size="20.0" />
+               </font>
+            </Label>
+         </children>
+      </HBox>
+   </top>
+   <center>
+      <HBox fillHeight="false" BorderPane.alignment="CENTER">
+         <children>
+            <Label fx:id="bankNumberLabel" text="0000 0000 0000">
+               <font>
+                  <Font size="20.0" />
+               </font>
+            </Label>
+         </children>
+         <BorderPane.margin>
+            <Insets bottom="30.0" top="50.0" />
+         </BorderPane.margin>
+      </HBox>
+   </center>
+   <padding>
+      <Insets bottom="20.0" left="20.0" right="20.0" top="20.0" />
+   </padding>
+   <bottom>
+      <HBox fillHeight="false" spacing="200.0" BorderPane.alignment="CENTER">
+         <children>
+            <VBox alignment="CENTER_LEFT" spacing="20.0">
+               <children>
+                  <Label text="CARD HOLDER">
+                     <font>
+                        <Font size="16.0" />
+                     </font>
+                  </Label>
+                  <Label fx:id="userNameLabel" text="Nguyen The Vinh">
+                     <font>
+                        <Font size="16.0" />
+                     </font>
+                  </Label>
+               </children>
+            </VBox>
+            <VBox alignment="CENTER_RIGHT" spacing="20.0">
+               <children>
+                  <Label text="VALID THRU">
+                     <font>
+                        <Font size="16.0" />
+                     </font>
+                  </Label>
+                  <Label text="01/08/2030">
+                     <font>
+                        <Font size="16.0" />
+                     </font>
+                  </Label>
+               </children>
+            </VBox>
+         </children>
+      </HBox>
+   </bottom>
+</fx:root>
Index: src/main/java/myApp/controllers/components/BankBox.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/myApp/controllers/components/BankBox.java b/src/main/java/myApp/controllers/components/BankBox.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/myApp/controllers/components/BankBox.java	
@@ -0,0 +1,27 @@
+package myApp.controllers.components;
+
+import javafx.fxml.FXMLLoader;
+import javafx.scene.layout.BorderPane;
+
+import java.awt.*;
+import java.io.IOException;
+
+public class BankBox extends BorderPane {
+    public Label bankNameLabel;
+    public Label bankNumberLabel;
+    public Label userNameLabel;
+    public BankBox() {
+//        FXMLLoader fxmlLoader = new FXMLLoader(getClass().getResource("/components/budgetBox.fxml"));
+//        fxmlLoader.setRoot(this);
+//        fxmlLoader.setController(this);
+//
+//        try {
+//            fxmlLoader.load();
+//            initialize(category, budget, spent, endDate, percentage, progressValue);
+//            updateButton.setOnAction(this::updateBudget);
+//            deleteButton.setOnAction(this::deleteBudget);
+//        } catch (IOException e) {
+//            throw new RuntimeException(e);
+//        }
+    }
+}
